portugues/revisa/tenta-consegue/reproduz.md

```diff
@@ -1,2 +1,4 @@
 Voc� envia uma resposta ao desenvolvedor descrevendo os passos para a reprodu��o do bug.
-Voc� aguarda ansiosamente pela resposta, e espera que nova vers�o sem o bug seja lan�ada logo.
\ No newline at end of file
+Voc� aguarda ansiosamente pela resposta, e espera que nova vers�o sem o bug seja lan�ada logo.
+
+[Ap�s um m�s sem obter resposta, voc� decide contatar novamente o desenvolvedor](sem-resposta/contato-desenvolvedor.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.